Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -339,15 +339,8 @@ protected Conditions createConditions(final CallbackProperties properties) {
             // Create resource for Authentication Decision Statement
             final String resource = properties.getAuthorizationResource();
 
-            // Options are Permit, Deny and Indeterminate
-            String decision = properties.getAuthorizationDecision();
-            if (decision == null) {
-                decision = AUTHZ_DECISION_PERMIT;
-            }
-
-            if (!isValidAuthorizationDecision(decision)) {
-                decision = AUTHZ_DECISION_PERMIT;
-            }
+            //"Decision.  The Decision attribute of the Authorization Decision Statement must be Permit" &CONN-1758
+            String decision = AUTHZ_DECISION_PERMIT;
```

Comment:
Unneeded local variable, just put the constant value into the method a couple lines below.